Dockerfile

```diff
@@ -80,23 +80,12 @@ COPY requirements/dev.txt /tmp/requirements.txt
 # Install development dependencies
 RUN pip install -r /tmp/requirements.txt
 
-# Install dockerize. It is used to ensure that the database service is accepting
```

Comment:
In your commit message:
 
> It appears more effective

Can you explicit why in the commit message?

Reply:
I think I'm explaining why in the commit message. Not sure to understand what you mean.